### PR TITLE
update vision FT examples corresponding to recent component update

### PR DESCRIPTION
**Pull request by shubhamiit**
# Description

classification- https://ml.azure.com/experiments/id/493baa1e-f330-4e86-adef-3d3f172c0d6d?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rupaljain-rs/providers/Microsoft.MachineLearningServices/workspaces/rupaljain-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

OD- https://ml.azure.com/experiments/id/294c311b-2e54-430b-8b29-b628cf55e3bd?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rupaljain-rs/providers/Microsoft.MachineLearningServices/workspaces/rupaljain-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

IS- https://ml.azure.com/experiments/id/24a6081a-b3d0-47f4-8e75-a2b1838ba04d?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rupaljain-rs/providers/Microsoft.MachineLearningServices/workspaces/rupaljain-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47
# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: cli/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification-pipeline.yaml
Comment (rjaincc) [inline cli/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification-pipeline.yaml:100]: Why comment the model eval args?

Comment (shubhamiit) [reply cli/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification-pipeline.yaml:100]: default value is label and image_url in pipeline component: https://github.com/Azure/azureml-assets/pull/1154/files

File: sdk/python/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification.ipynb
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification.ipynb:175]: I would create a cell with model eval cluster creation. There I will use the same name as that of FT cluster.

**Review by rjaincc — APPROVED** [review]
Please link the runs. 
Approving with minor suggestion on ME cluster. 
Let's ensure to not merge the PR until our components are released.

**Review by gauravrajguru — APPROVED** [review]

**Pull request merged**
